activesupport/lib/active_support/core_ext/module/attribute_accessors_per_thread.rb

```diff
@@ -118,6 +118,8 @@ def #{sym}=(obj)
   #   class Customer < Account
   #   end
   #
+  #   Account.user = 'DHH'
```

Comment:
@matthewd This was added in support of the above mentioned line `if the parent class changes the value, the value of subclasses is not changed.`

The context of presence of `Account` class is taken from where you are suggesting, but here I am reassigning the variable after inheritance.